### ISSUE TITLE
Test failures on Windows

### ISSUE THREAD

**Issue by nasmdh** [eclipse/xtext-web#42]
Hello,

Could you please have a look on this error. I am building a fresh cloned source. Thank you.

68 tests completed, 35 failed
:org.eclipse.xtext.web:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':org.eclipse.xtext.web:test'.
> There were failing tests. See the report at: file:///C:/Users/****/Git/xtext-web3/org.eclipse.xtext.web/build/reports/tests/test/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2 mins 42.843 secs

**Comment by cdietrich** [comment]
Can you try the gradle build with `-PuseJenkinsSnapshots=true`

**Comment by cdietrich** [comment]
can you oopen that test report and see what fails?

**Comment by cdietrich** [comment]
looks like all the tests are made not to work on windows.
you can build with `-x test` to skip the tests

**Comment by cdietrich** [comment]
sample error

```
org.junit.ComparisonFailure: expected:<FormattingResult [[
  stateId = "-7fffffff"
  formattedText = "/* bla */ output signal x\nstate foo\n  set x = true\nend\n"
]
]> but was:<FormattingResult [[
  stateId = "-7fffffff"
  formattedText = "/* bla */ output signal x\r\nstate foo\r\n  set x = true\r\nend\r\n"]
]>
```

**Comment by cdietrich** [comment]
@spoenemann what are the services like formatter service intended to produce
`\n` or `\r\n` ?

**Comment by nasmdh** [comment]
thank you Christian for the quick response.

I tried the gradle, it results the same error.

Best,
Nesredin



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by cdietrich** [comment]
Fixed in Xtext 2.13
